### ISSUE TITLE
SplatBlend_Shared.cginc causes Crash //Unity Version 5.61b

### ISSUE THREAD

**Issue by Gblur** [slipster216/VertexPaint#20]
Got the Issue that especially this file can't get imported.
unfortunately I cant give you more informations.
I have no idea were to debug.

**Comment by slipster216** [comment]
I would suggest removing the shader variants your not using. It seems Unity
changed the shader compiler- most likely it is producing more variants for
the new lighting system - and this is pushing the amount of compiling the
system does over the limit of what their compiler will do and causes it to
crash.

**Comment by Gblur** [comment]
Ok It works. Importing only 1 Shader takes a long time. But problem solved.
Thx

**Comment by slipster216** [comment]
Yeah, it really pushes the limits on unity's shader compiler; that's why I
ended up moving to a shader generator for MegaSplat..
